### PR TITLE
add extra feature "choosenim show path"

### LINKED ISSUE #260: "choosenim show path" command
Some projects (like [napibindings](https://github.com/AjBreidenbach/napibindings)) need a command that prints only the path of the current Nim version.

```
$ choosenim show path
/Users/foo/.choosenim/toolchains/nim-1.4.8
```

### PR DESCRIPTION
**Pull request by ilmanzo**
fix #260

### CODE REVIEW

File: src/choosenim.nim [hunk 1]
```diff
@@ -207,6 +207,11 @@ proc show(params: CliParams) =
   let channel = getCurrentChannel(params)
   let path = getSelectedPath(params)
   let (_, version) = getNameVersion(path)
+  if params.commands.len==2:
```
Comment (dom96) [inline src/choosenim.nim:210]:
```suggestion
  if params.commands.len == 2:
```

File: src/choosenim.nim [hunk 2]
```diff
@@ -207,6 +207,11 @@ proc show(params: CliParams) =
...
+    let whatToShow=params.commands[1]
```
Comment (dom96) [inline src/choosenim.nim:211]:
```suggestion
    let whatToShow = params.commands[1]
```

File: src/choosenim.nim [hunk 3]
```diff
@@ -207,6 +207,11 @@ proc show(params: CliParams) =
...
+    if whatToShow.toLowerAscii()=="path":
```
Comment (dom96) [inline src/choosenim.nim:212]:
```suggestion
    if whatToShow.toLowerAscii() == "path":
```

**Review by dom96 — APPROVED** [review]
Nice, thanks! If you have time for another PR: a test for this would be awesome.

**Comment by ilmanzo** [comment]
> Nice, thanks! If you have time for another PR: a test for this would be awesome.

sure, please take a look at #276

**Pull request merged**